extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -144,10 +146,10 @@ oppia.factory('ItemSelectionInputValidationService', [
               });
             });
           });
-          areAllChoicesCovered = handledAnswers.every(function(handledAnswer) {
-            return handledAnswer;
-          });
         }
+        areAllChoicesCovered = handledAnswers.every(function(handledAnswer) {
```

Comment:
Why has this moved from inside the { ... } to outside? (Look at the diff in the "files changed" tab.)

Reply:
I think I took it out when I was trying to extend it for isProperSubset, (including all cases of maxAllowedCount). Shall I put it back in?

Reply:
What is the correct thing to do? Please analyze.

Reply:
If we are not going to extend it later, (I thought of doing it for all functions in the other issue, but I saw there isnt much use of that, and it would make the code more messy)
I will add it back inside the brackets. I will do it now 
Thanks!